onnxruntime/test/providers/cpu/math/matmul_test.cc

```diff
@@ -179,6 +179,10 @@ void RunMatMulTest(int32_t opset_version) {
 }
 
 TEST(MathOpTest, MatMulFloatType) {
+  // TODO: Unskip when fixed
+  if (DefaultDmlExecutionProvider().get() != nullptr) {
+    GTEST_SKIP() << "Skipping because of the following error: Assertion failed: m_bufferTensorDesc.TotalTensorSizeInBytes >= ComputeByteSizeFromDimensions(nonBroadcastDimensions, dataType)";
```

Comment:
Is the error message reversed? Should it say:

`m_bufferTensorDesc.TotalTensorSizeInBytes < ComputeByteSizeFromDimensions...`?

Because it should be okay for the padded/aligned tensor byte size to be bigger than the minimal tensor size.

Reply:
Never mind. I see where the assert comes from now.

```
    m_bufferTensorDesc.TotalTensorSizeInBytes = DMLCalcBufferTensorSize(
        m_bufferTensorDesc.DataType,
        m_bufferTensorDesc.DimensionCount,
        m_sizes,
        useStrides ? m_strides : nullptr
        );
    assert(m_bufferTensorDesc.TotalTensorSizeInBytes >= ComputeByteSizeFromDimensions(nonBroadcastDimensions, dataType));
```